### DISCUSSION TITLE
SuperChlorinate not working

### DISCUSSION THREAD

**Discussion by dnikles** [tagyoureit/nodejs-poolController#803] [Q&A]
The super chlorinate commands don't seem to do anything for me. I don't get any errors from the server, but wether I do it through dash panel or the API my chlorinator doesn't seem to change what it's doing (if I put it to superhchlorinate, I don't see the lights go on like it's super chlorinating, if I try to cancel, I don't see the lights go back down to whatever the chlorinator level is set at). I've attached a packet capture.
[poolreplay.zip](https://github.com/tagyoureit/nodejs-poolController/files/11900019/poolreplay.zip)

**Comment by dcorrea78** [comment]
I'm having the same issue now as well, and the chlorinator is not responding to inputs. For example, if I increase the output the Gui will reflect the increase but the easy touch system does not change. 

All other functions work. I know this worked as recently as last month so I'm not sure what changed. Logs attached.

[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/11926716/replay.zip)

**Comment by dnikles** [comment]
I rolled back to 7.7 and it’s working for me in that version.

**Reply by dcorrea78** [reply]
What's the best way to rollback?

**Reply by tagyoureit** [reply]
Can one of you provide the working 7.7 (compared to 8.0)?  One of us will take a look.

**Reply by CodeSlinger69** [reply]
When you cloned the repo, you got all the previous versions as well, so all you need to do is this.

Show the tags (versions) that are available.
`$ git tag -l`

`5.3.0
6.0.0
7.1.0
v3.1.13
v3.1.9
v4.0
v4.0.0
v5.0.0
v5.1.1
v5.2.0
v5.3.0
v6.1.0
v6.5.1
v7.0.1
v7.2.0
v7.3.1
v7.4.0
v7.6.0
v7.6.1
v8.0.0
v8.0.1
v8.0.2
v8.0.3
`

Then Checkout (use) a specific tag:
`$ git checkout tags/<tag_name>`

Unfortunately, 7.7.0 doesn't seem to be tagged. You can try v7.6.1 and see if it works.

**Reply by CodeSlinger69** [reply]
But, I did have 7.7.0 snapshotted in a VM :)  Here you go.

[njspc-7.7.0.tar.gz](https://github.com/tagyoureit/nodejs-poolController/files/11928575/njspc-7.7.0.tar.gz)

**Comment by CodeSlinger69** [comment]
I had this same problem about a month ago, and it pushed me to build a Nixie. Not saying you should do that, but if you are interested, the post is [here](https://github.com/tagyoureit/nodejs-poolController/discussions/801).

**Comment by dnikles** [comment]
> Can one of you provide the working 7.7 (compared to 8.0)? One of us will take a look.

I apologize for the late response. I gave bad info earlier, I actually rolled back to 7.6.1. I am running the docker image from https://hub.docker.com/r/msmi/nodejs-poolcontroller . When you ask for the working version, do you mean you want the logs from that version?
